mozartdata/transforms/staging/yotpo_redemptions_kpi_aggregation.sql

```diff
@@ -0,0 +1,11 @@
+select
+    day::date as date
```

Comment:
```suggestion
    day::date as event_date
```
I know you changed these to `event_date` in the query, but let's make our staging tables follow a better practice - call these `event_date`. Totally know I'm at fault for some tech debt on other tables, but you can be better than me.